files/default/sql/ddl/2.2.0__initial_tables.sql

```diff
@@ -370,16 +370,19 @@ CREATE TABLE `feature_store` (
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `feature_store_statistic` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `commit_time` VARCHAR(20) COLLATE latin1_general_cs NOT NULL,
+  `commit_time` DATETIME(3) NOT NULL,
   `inode_pid` BIGINT(20) NOT NULL,
   `inode_name` VARCHAR(255) COLLATE latin1_general_cs NOT NULL,
   `partition_id` BIGINT(20) NOT NULL,
   `feature_group_id` INT(11),
+  `feature_group_commit_id` BIGINT(20),
   `training_dataset_id`INT(11),
   PRIMARY KEY (`id`),
   KEY `feature_group_id` (`feature_group_id`),
   KEY `training_dataset_id` (`training_dataset_id`),
+  KEY `feature_group_commit_id_fk` (`feature_group_id`, `feature_group_commit_id`),
   CONSTRAINT `fg_fk_fss` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
+  CONSTRAINT `fg_ci_fk_fss` FOREIGN KEY (`feature_group_id`, `feature_group_commit_id`) REFERENCES `feature_group_commit` (`feature_group_id`, `commit_id`) ON DELETE CASCADE ON UPDATE NO ACTION,
```

Comment:
I would not add the `on delete cascade` here, but rather `on delete set null`, the reason is that you might want to compact the commits, but you don't want to lose the statistics.